### ISSUE TITLE
Issue going to destination

### ISSUE THREAD

**Issue by tjb8907** [Stephan-S/AutoDrive#3]
I am really starting to like this mod more and more! It take a bit to record everything, but it coupled with courseplay is awesome. I can send tractors to fields and then let courseplay run the field route, and then use Autodrive to send them back to the farm. I like AutoDrive better than course play because you don't have to record a separate route all the way from the farm to field each time. (you can do it with courseplay too, but this is easier I think). 

Anyway, I have adapted the config file to work with my custom map, and it is working good for some things, but I am having a problem with some points/destinations that I have saved. Some of the earlier markers that I created work good, the vehicle will drive to them and stop when the reach it, but now some of don't work. I can select the marker and start it, but it will just drive to the next closest waypoint, stop, and say it has reached the destination. 

I realize you probably really only support GoldCrest map, but I know it works on my map for some markers. Is it something I am doing wrong? The markers are connected to routes properly, as I don't think you can even engage autoDrive if there isn't a valid course to that marker right?

**Comment by Stephan-S** [comment]
Well, first of all im glad you like the mod so far :-)

Are you certain that you recalculated the routes again after recording new waypoints and destinations? Are you 100% sure the destinations are connected everywhere. Especially if you have connected them in the correct direction. 

Sometimes it might help to restart the game so that AutoDrive reloads everything from the config file again and has no issues with deleted markers or uncalculated routes.

But yes, you can activate AutoDrive if the destination is not connected but as far as I can remember my implementation it should not start driving and print a message to the log like "AutoDrive hat ein Problem beim Initialisiseren festgestellt" which means it did not find a suitable path.

If you would be so kind as to send me the log file and maybe even your config file so I can locate the error. If it`s in my code or in your recordings

**Comment by tjb8907** [comment]
I think I have them all connected right, but I will check. When connecting two routes, the blue should be on the previous point and white on the next point going forward correct? That is how I did it on most and that seemed to work for a while. I did record some of the routes with v0.8.5 if that makes a difference, now I am on v1.0.2. I will double check my recordings and get you the log when I get home.
